core/src/main/resources/lib/form/submit.jelly

```diff
@@ -50,7 +50,7 @@ THE SOFTWARE.
     <s:attribute name="clazz" />
   </s:documentation>
 
-  <button id="${attrs.id}" name="${attrs.name ?: 'Submit'}"
+  <button id="${attrs.id}" name="${attrs.name ?: 'Submit'}" formNoValidate="true"
```

Comment:
The examples I've read seem to consistently use `formnovalidate` rather than `formnovalidate="true"`.  Since it iis described as a boolean, it seems safe to use ``true`.

Is there another reason for the selected syntax rather than setting the attribute `formnovalidate` without assigning a value?
```suggestion
  <button id="${attrs.id}" name="${attrs.name ?: 'Submit'}" formNoValidate="true"
```

Reply:
Jelly is XML and doesn't support value-less attributes.

Reply:
I was about to answer like @daniel-beck about the impossibility to have value-less attributes, but I didn't have in mind at all about:

> the value should be an empty string, or match the attribute name

Thanks a lot for the information @daniel-beck, I commited your suggestion after testing locally that it was working.

Reply:
👍